dask_ml/preprocessing/data.py

```diff
@@ -47,7 +50,9 @@ class StandardScaler(sklearn.preprocessing.StandardScaler):
 
     __doc__ = sklearn.preprocessing.StandardScaler.__doc__
 
-    def fit(self, X, y=None):
+    def fit(
+        self, X: Union[ArrayLike, DataFrameType], y: Optional[SeriesType] = None
```

Comment:
Why just SeriesType here (and other places) for `y`? It can also be an array I think.